### ISSUE TITLE
select which T1 to use when multiple _T1w.nii.gz are present

### ISSUE THREAD

**Issue by bpinsard** [poldracklab/fmriprep#1767]
For instance if you acquire in the same session a brain T1w which outputs `..._T1w.nii.gz` and `..._rec-norm_T1w.nii.gz` (scanner normalization) and also a cervical spinal cord T1w named `....loc-spine_T1w.nii.gz`, fmriprep seems to try making an average of all these. 
Is there an option to select from a BIDS anat directory, the specific T1w to select for anatomical processing using a glob or else?
Thanks!

**Comment by oesteban** [comment]
Yes, this is indeed a problematic issue. I'm transferring this issue over to poldracklab/smriprep though, because the anatomical processing has been outsourced.
